hack/debug.sh

```diff
@@ -73,7 +89,13 @@ do_master () {
     mkdir -p $logmaster
 
     # Log the generic system stuff
-    log_system $logmaster $aos_master_service
+    log_services=
+    for service in "${services[@]}"; do
+        if [ -n "${service}" ]; then
+            log_services="${log_services} ${service}"
+        fi
+    done
```

Comment:
My bash is not the best, as you've found.  ${services[*]} would certainly be easier.